release-notes/swan-lake-beta2-release-note.md

```diff
@@ -52,10 +52,33 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github
 
 ### Standard Library Updates
 
+The new PostgreSQL database package is introduced with this release. This package provides the functionality required to access and manipulate data stored in a PostgreSQL database.
+
 #### New Features
 
+##### Log Package
+Added Observability span context values to log messages when Observability is enabled.
+
+##### I/O Package
+Introduced `io:fprint` and `io:fprintln` APIs.
+```ballerina
+io:fprint(io:stderr, "Unexpected error occurred");
+io:fprintln(io:stderr, "Unexpected error occurred");
+io:fprint(io:stdout, "Passed without an error");
+io:fprintln(io:stdout, "Passed without an error");
+```
+
 #### Improvements
 
+##### Database Packages
+Returned type of the stream is now inferred as a second parameter to the query remote method.
+For more info see [this issue](https://github.com/ballerina-platform/ballerina-standard-library/issues/1445)
```

Comment:
```suggestion
For more info, see [this issue](Processing streams in database operations are bit confusing).
```
normally, we do not link to issues like this in the Release Note.

Reply:
@lahirusamith Can you check this?

Reply:
@kalaiyarasiganeshalingam, as we discussed, shall we add a description here rather than an issue link. ?